### ISSUE TITLE
slidev

### ISSUE THREAD

**Issue by aarmn** [NixOS/nixpkgs#198297]
**Project description**
Presentation Slides for Developers 

**Metadata**
It's a npm package, I guess that would make it easier to package, but its a tool, not a lib, so i think it should be package like poetry directly under nixpkgs and not nodePackages

* homepage URL: https://sli.dev/
* source URL: https://github.com/slidevjs/slidev
* license: mit
* platforms: linux, darwin

**Comment by hbjydev** [comment]
+1 for this one, it'd be great

**Comment by hbjydev** [comment]
Although @aarmn I think it should be under nodePackages, because there are already plenty of tools in there, like `tailwindcss` and a lot of the JS-based LSP servers

**Comment by aarmn** [comment]
> 

Yup, but it's a non-dev, non-lib utility, so the case is similar to poetry pkg for python, It's end-user enough to doesn't matter its node and make sense not to be there, but I let the final decision be taken by the people who take the task and do it

**Comment by Laurent2916** [comment]
+1, but since the project uses `pnpm`, we have to wait for #231513

**Comment by usmcamp0811** [comment]
I haven't been succesful in using `yarn2nix` with `slidev` yet.. and I am no JS expert but when I do something simple like:

```
npm init slidev
```

It gives you an option to choose `npm`, `yarn`, or `pnpm`. 

So do we still need to wait on #231513 ?

**Comment by dbrw** [comment]
Now that  #231513 been solved, how can we make it happen?

**Comment by usmcamp0811** [comment]
This guy seemed to have figured it out --> https://github.com/charles-bord/nix-forest-slides

I just ported it over to [my dotfiles](https://gitlab.com/usmcamp0811/dotfiles) as I am trying to throw together some slides for a demo..

**Comment by usmcamp0811** [comment]
I've been working with Sli.dev for the past week and I have got a couple versions in my dotfiles repo, and I patched `prettier` with the `slidev` plugin. I can't get any of the newer versions of Slidev built because they are using a patched lock file and I am too ignorant in the ways of the front end to know how to freeze the lock file.
